spark-icons/build.gradle.kts

```diff
@@ -41,4 +41,6 @@ kotlin {
 dependencies {
     implementation(libs.androidx.compose.ui)
     implementation(libs.androidx.appCompat.resources) // Needed for compat vector drawables
+    api(libs.androidx.compose.animation.graphics) // needed for the animated icons
```

Comment:
FYI there is a special construct for adding comments in code :)

```kotlin
api(libs.androidx.compose.animation.graphics) {
  because("needed for the animated icons")
}
```

Reply:
Which API type/class is needed?

Reply:
Nice I knew a syntax existed but couldn't remember what it was!